modules/manager/run.go

```diff
@@ -135,7 +135,6 @@ func Run(
 	cfg Config,
 	scheme *runtime.Scheme,
 	setupControllers SetupControllersFunc,
-	admissionRequestHandler AdmissionRequestHandlerFunc,
 	startedChan chan<- struct{},
```

Comment:
Could we mention that we  removed the param `admissionRequestHandler` since we have removed admission webhooks in the PR comments?

Reply:
Mentioned in the PR description 👍